.openshiftio/application.yaml

```diff
@@ -25,30 +33,34 @@ parameters:
   from: '[a-zA-Z0-9]{40}'
   generate: expression
 objects:
-- apiVersion: v1
+- apiVersion: image.openshift.io/v1
```

Comment:
Shouldn't `apiVersion`s of other resources be updated as well?

Reply:
Yaa, it should be updated but It will not affect. That's like good to have.

Reply:
actually if we make this change or not this does not make any difference, I will revert this one.